src/test/java/unit/uk/gov/ida/verifyserviceprovider/services/EidasAssertionTranslatorTest.java

```diff
@@ -98,12 +98,13 @@ public void setUp() {
             metadataResolverRepository,
             signatureValidatorFactory,
             HUB_CONNECTOR_ENTITY_ID,
+            singletonList(HUB_CONNECTOR_ENTITY_ID),
             userIdHashFactory);
         doNothing().when(instantValidator).validate(any(), any());
         doNothing().when(subjectValidator).validate(any(), any());
         doNothing().when(conditionsValidator).validate(any(), any());
         doNothing().when(levelOfAssuranceValidator).validate(any(), any());
-        when(metadataResolverRepository.getResolverEntityIds()).thenReturn(asList(STUB_COUNTRY_ONE));
+        when(metadataResolverRepository.getResolverEntityIds()).thenReturn(singletonList(STUB_COUNTRY_ONE));
```

Comment:
What's the benefit of a singletonList here?

Reply:
IntelliJ highlighted it and offered singletonList. I've not yet explored singletonList, but Array.asList doesn't actually make a List. It makes something that implements List but throws exceptions if you try to add/remove items - which I do have feelings about.

Reply:
Off the top of my head, I think it creates an immutable list (which is why it bogs out when you try to add or remove from the list).